ldaptive/src/main/java/org/ccci/idm/user/ldaptive/dao/io/GroupValueTranscoder.java

```diff
@@ -50,47 +67,27 @@ public void setNameRdnAttr(@Nonnull final String rdnAttr) {}
 
     @Override
     public String encodeStringValue(@Nonnull final Group group) {
-        final StringBuilder sb = new StringBuilder();
-
-        // append components
-        for (final Dn.Component component : Lists.reverse(group.getComponents())) {
-            if(sb.length() > 0) {
-                sb.append(delimiter);
-            }
-            sb.append(component.type).append(valueDelimiter).append(LdapAttribute.escapeValue(component.value));
-        }
-
-        if (baseDn.length() > 0) {
-            sb.append(delimiter).append(this.baseDn);
-        }
+        final StringBuilder sb = new StringBuilder(DnUtils.toString(group));
```

Comment:
Could reduce to `return (new StringBuilder(DnUtils.toString(group))).toString();`